packages/nouns-contracts/contracts/governance/NounsDAOLogicV2.sol

```diff
@@ -97,6 +97,9 @@ contract NounsDAOLogicV2 is NounsDAOStorageV2, NounsDAOEventsV2 {
     /// @notice The vote refund gas overhead, including 7K for ETH transfer and 29K for general transaction overhead
     uint256 public constant REFUND_BASE_GAS = 36000;
 
+    /// @notice The maximum gas units the DAO will refund voters on
+    uint256 public constant MAX_REFUND_GAS_USED = 200_000;
```

Comment:
can you add to the documentation how you decided on the value of 200K?
how long of a reason text does that cover?

Reply:
it covers the reason text in the tests, which is 2 times the longest reason I found (from honk).
decided on this value by seeing how much that text got me to (about 220K gas) then cut back a bit.

honk's reason was 882 words, so this supports about 1764 words.
I'm thinking reason strings shouldn't be extremely long (as opposed to proposal descriptions that can get quite long)

wdyt?

Reply:
I'll add to the natspec a specific number of words / characters this limit supports

Reply:
fixed